src/modules/auth/auth.controller.ts

```diff
@@ -69,22 +71,12 @@ export class AuthController {
   }
 
   @Get('refresh')
+  @UseGuards(RefreshTokenGuard)
   async refresh(
     @Req() req: Request,
```

Comment:
I think you there is a decorator GetUser in the project that you can use to get req.user like this for example:
```js
@GetUser() payload: JwtPayload
```

Reply:
fixed.